DO NOT MERGE [Workers KV] KV product tile (#9656)
cloudflare/cloudflare-docs: content/kv/api/delete-key-value-pairs.md

---
pcx_content_type: concept
title: Delete key-value pairs
weight: 7
---

# Delete key-value pairs

To delete a key-value pair, call the `delete()` method on any KV namespace you have bound to your Worker code:

```js
await NAMESPACE.delete(key);
```

> Calling the `delete()` method will remove the key and value from your KV namespace. As with any operations, it may take some time to see that the key has been deleted from various points in the Cloudflare global network.

Reviewer: ```suggestion
Calling the `delete()` method will remove the key and value from your KV namespace. As with any operations, it may take some time for the key to be deleted from various points in the Cloudflare global network.
```

Reviewer: My suggestion here is mainly concerning inclusion, as not all our users are able to "see". If this changes too much the meaning of the sentence, feel free to ignore
